From 78eab552a4076f3d50233d00496b9f22e7f2168e Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.corp-partner.google.com>
Date: Mon, 8 Jun 2026 17:31:31 +0000
Subject: [PATCH] Remove redundant definition of FORTIFY_SOURCE

Remove the redundant -D_FORTIFY_SOURCE since Android's
build system already sets the appropriate fortify level

Tracked-On: NA
Signed-off-by: Tekriwal, Tanuj <tanuj.tekriwal@intel.corp-partner.google.com>
---
 Android.bp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Android.bp b/Android.bp
index 448dce5..f2d6a82 100644
--- a/Android.bp
+++ b/Android.bp
@@ -18,7 +18,6 @@ cc_library_static {
 
     cflags: [
         "-Wno-int-to-pointer-cast",
-        "-D_FORTIFY_SOURCE=2",
         "-fstack-protector-strong",
         "-fno-delete-null-pointer-checks",
         "-fwrapv",
-- 
2.34.1

